packages/financial-templates-lib/src/proxy-transaction-handler/DSProxyManager.js

```diff
@@ -1,5 +1,6 @@
-const { getFromBlock } = require("@uma/common");
+const assert = require("assert");
```

Comment:
assert was previously incorrectly working as this was only tested in unit tests.